packages/libraries/theming/size/ods-theming-size.breadcrumb.scss

```diff
@@ -0,0 +1,13 @@
+/// @access private
+@function ods-get-breadcrumb-size-properties($size-name) {
+  @return (
+    padding: var(--ods-size-breadcrumb-#{$size-name}-padding)
+  )
+}
+
+/// @access private
```

Comment:
wasn't there something we could do so that the IDE's interpretation of /// wouldn't be broken like this?

Reply:
no idea TBH